backends/python-fastapi/app/usecase/gate/interactor.py

```
@@ -61,6 +65,11 @@ def issue_token(self, dto: GateIssueDto) -> GateIssueVo:
 
         token = self._issue_jwt(client.private_key, client.identifier, dto.member)
         self.cache_repo.put_jwt(client.identifier, dto.member, token)
+        if self.history_repo and client.id:
+            try:
+                self.history_repo.save(client.id, dto.member, datetime.now(), token)
+            except Exception:
+                pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)